dist/tools/vagrant/README.md

```diff
@@ -42,15 +42,19 @@ vagrant provision
 This will re-run the [provisioning script](https://github.com/RIOT-OS/RIOT/tree/master/dist/tools/vagrant/bootstrap.sh).
 
 ## Inside the VM
-Once logged in to the VM via `vagrant ssh` you can find the RIOT-OS root directory in your
-working directory. This is a shared directory and stays synchronized with your RIOT-OS directory
+Once logged in to the VM via `vagrant ssh` you can find the RIOT root directory in your
+working directory. This is a shared directory and stays synchronized with your RIOT directory
 on the host system. All changes made will be mirrored from the host system to the guest system
 and vice versa.
 
-This feature allows you to conveniently develop code for RIOT-OS with your preferred IDE on
-your host system and use the VM for compiling, flashing devices and running the native port of RIOT-OS.
+This feature allows you to conveniently develop code for RIOT with your preferred IDE on
+your host system and use the VM for compiling, flashing devices and running the native port of RIOT.
 
 ## Additional Information
 For new boards it is necessary to add new udev rules to `dist/tools/vagrant/udev_rules` and
 additional usb filters to the Vagrantfile so that VirtualBox is able to capture the devices.
 The needed `vendor id` and `product id` can be obtained by running `vboxmanage list usbhost`.
+
+### Virtualbox (for Linux Users)
```

Comment:
I don't like standalone subsubsection, their should be at least 2 or none. Improvement: change structure below _Additional Information_ into a item/bullet list?